core-libs/setup/tsconfig.spec.json

```diff
@@ -1,12 +1,404 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "baseUrl": "./",
     "outDir": "../../out-tsc/spec",
     "strict": false,
-    "target": "es2020",
-    "module": "es2020",
-    "types": ["jasmine", "node"]
+    "module": "CommonJs",
+    "types": ["jest"],
+    "paths": {
```

Comment:
They are necessary in jest setups. Otherwise, the [import](https://github.com/SAP/spartacus/pull/16532/files#diff-a8e35d359e755e7aa63e750b9b96184549bc9fd60c820cf986bcbfa178491c6bR9) from `@spartacus/core` wouldn't work in the setup's test.

Reply:
Wouldn't it suffice to add just the path mapping only for `@spartacus/core`?
Or to extend some ts-config where all those mapping mappings are already defined? 

My intention is not to repeat ourselves so much with those mapping.